src/features/island/fisherman/FishingChallenge.tsx

```diff
@@ -0,0 +1,339 @@
+        <ResizableBar
+          percentage={100 - (hits / health) * 100}
+          type={"error"}
+          outerDimensions={{
+            width: 50,
+            height: 8,
+          }}
+        />
+      </div>
+      {/* Bar */}
+      <div
+        className="mx-auto my-2 relative pointer-events-none"
+        id="fishing-ring"
+        style={{
+          width: "200px",
+          height: "200px",
+        }}
+      >
+        <div
+          className="w-full h-full"
+          style={{
+            border: "20px solid #193c3e",
+            borderRadius: "50%",
+            outline: "3px solid #fff",
+            boxSizing: "border-box", // Include the border in the element's dimensions
+            boxShadow: "inset 0 0 0 3px #fff", // Create an inset box shadow
+          }}
+        />
+        <animated.svg
+          width="200"
+          height="200"
+          viewBox="0 0 200 200"
+          id="fishing-bar"
+          className="absolute inset-0 "
+          style={greenBarProps}
+          // style={{
+          //   animation:
+          //     state === "playing" ? `spin ${speed}s linear infinite` : "none",
+          // }}
+        >
+          <circle
+            cx="100"
+            cy="100"
+            r="90"
+            fill="none"
+            stroke="#193c3e"
+            strokeWidth="12"
+          />
+          <circle
+            strokeDasharray={`${barHeight} ${100 - barHeight}`}
+            strokeDashoffset={25 + barHeight / 2}
+            cx="100"
+            cy="100"
+            r="90"
+            fill="none"
+            stroke="#63c74d"
+            strokeWidth="12"
+            pathLength="100"
+          />
+        </animated.svg>
+
+        <svg
+          width="200"
+          height="200"
+          viewBox="0 0 200 200"
+          className="absolute inset-0 "
+          style={{
+            transform: `rotate(${castFrom}deg)`,
+          }}
+        >
+          <circle
+            strokeDasharray={`${0.5} ${100 - 0.5}`}
+            strokeDashoffset={25 + 0.5 / 2}
+            cx="100"
+            cy="100"
+            r="95"
+            fill="none"
+            stroke={castIndicatorColour}
+            strokeWidth="12"
+            pathLength="100"
+            strokeOpacity={castIndicatorColour === "#ffffff" ? 0.6 : 1}
+          />
+        </svg>
+
+        <animated.div
+          id="tentacle"
+          style={{
+            position: "absolute",
+            width: "20px",
+            height: "100px",
+            top: "0px",
+            left: "90px",
+            transformOrigin: "bottom center",
+            // transform: `rotate(${tentacleAngle}deg)`,
+            ...tentacleProps,
+          }}
+        >
+          <img
+            src={ringDot}
+            style={{
+              position: "absolute",
+              width: "24px",
+              top: "1px",
+              left: "2px",
+              rotate: "137deg",
+            }}
+          />
```

Comment:
```suggestion
          <img
            src={ringDot}
            style={{
              position: "absolute",
              width: "24px",
              top: "1px",
              left: "-1px",
              rotate: "137deg",
            }}
          />
```

This appears to be off center from the parent div - which gives a bias when the green bar is rotating counter clockwise

![bias](https://github.com/sunflower-land/sunflower-land/assets/41215134/bd0c953f-9aa1-4995-9cee-d363338de638)